templates/symfony-6.1.template.yaml

```diff
@@ -18,11 +18,11 @@ info:
       content: |
           Symfony 6.1<br />
           The webapp package that includes most of the needed features for developing web applications<br />
-          PHP 8.1<br />
+          PHP 8.2<br />
           PostgreSQL 13<br />
```

Comment:
Just asking, is version 13 correct?

Reply:
Good catch :) Fixed in 7e3855a84ffecabf9c7173a3d7ea6b394b916083